src/api/db/migrate/20190415010809_create_watched_items.rb

```diff
@@ -1,6 +1,7 @@
 class CreateWatchedItems < ActiveRecord::Migration[5.2]
   def change
     create_table :watched_items, id: :integer do |t|
+      t.belongs_to :user
```

Comment:
OBS is using integer for ids. That's because it was already around when Rails switched to bigint as default.
So will have to add `type: :integer` here.

Reply:
got it 👍